src/app/app.router.ts

```diff
@@ -2,9 +2,10 @@ import { ModuleWithProviders }  from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
 
 import { HomeComponent } from './components/home/home.component';
-import { EndpointListComponent } from './components/endpoint-list/endpoint-list.component';
-import { EndpointDetailsComponent } from './components/endpoint-list/endpoint-details/endpoint-details.component'
-import { JobDetailsComponent } from './components/endpoint-list/job-details/job-details.component'
+import { JobsWrapperComponent } from '@components/jobs-wrapper/jobs-wrapper.component'
+import { EndpointListComponent } from '@components/jobs-wrapper/endpoint-list/endpoint-list.component';
+import { EndpointDetailsComponent } from '@components/jobs-wrapper/endpoint-details/endpoint-details.component'
+import { JobDetailsComponent } from '@components/jobs-wrapper/job-details/job-details.component'
```

Comment:
I think to put semicolon in all places is a good practice and helps to avoid errors